app/controllers/users_controller.rb

```diff
@@ -0,0 +1,57 @@
+class UsersController < ApplicationController
+
+  before_action :ensure_current_user, except: [:new, :create, :show]
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
This is never used in a template / view - make it a local variable rather than an instance variable.